src/test/java/Test.java

```diff
@@ -0,0 +1,18 @@
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class Test {
+    @org.junit.Test
+    public void test() {
```

Comment:
Either delete this or better yet create a function with this logic so that we can check this in the future

Reply:
yes, I didn't mean to leave it in.